packages/cosmos_ui_components/lib/components/cosmos_circle_text_button.dart

```diff
@@ -5,32 +5,40 @@ class CosmosCircleTextButton extends StatelessWidget {
   final String text;
   final VoidCallback onTap;
   final IconData icon;
+  final bool isEnabled;
```

Comment:
to control enabled/disabled state lets follow flutter's approach, where setting `onTap` to null makes the button disabled

Reply:
Updated